### PR TITLE
CHOGM Conference Page: Create Dynamic Conference Page

### PR DESCRIPTION
**Pull request by peterdtitan**
I have created a dynamic site that makes use of JavaScript objects to load content as well as the local storage for dummy user login and sign-up functions. The following files and folders were added: 

- index.html (entry point for app)
- js folder (contains static data and script for functionality)
- css folder (relevant styling and accessibility features)
- images folder (contains items used in the website content sections)

[Watch The Presentation](https://drive.google.com/file/d/1R6O4u2spAG6TxzmDQkO9P-z_2jAFJCt4/view?usp=sharing)

Screenshot 2022-06-25 at 12 55 34 PM

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,76 @@
...
+## Acknowledgments
+
+- freeCodeCamp for implementation of modals
+- Rwanda CHOGM 2022 Event
+
+## 📝 License
+
+This project is [MIT](./MIT.md) licensed.
+
+### Original design idea by **Cindy Shin in Behance**, in association with the Creative Commons [license](./CC.md) of the design.
```
Comment (zainsadaqat) [inline README.md:76]: [Optional]
Please add this line under the "Acknowledgments" section 👍🏻

**Review by zainsadaqat — CHANGES_REQUESTED** [review]
# Required Changes 🔁
# Hello @peterdtitan  😄 

You're doing a great job on your project. We're proud of you. Good job so far! 

There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
  ✅  Correct use of Github Flow
  ✅  No linter errors
  ✅  Descriptive Pull Request

## Required Changes ♻️

- [x] Please add the checkbox pattern image like the required design behind the images 👍🏻 
#### [YOUR DESIGN]
Screen Shot 2022-06-25 at 3 14 02 PM
#### [REQUIRED DESIGN]
Screen Shot 2022-06-25 at 3 15 48 PM

- [x] In the upcoming events section, Please make the description look consistent. You may do this by reducing the title's font size 👍🏻 
Screen Shot 2022-06-25 at 3 21 17 PM

#### [OPTIONAL] 
You have done an extraordinary job by adding the sign-in and sign-up features. It will be good if a person is not signed in then it just shows that popup only once but it's the popup again and again that may not be good for some people. You can work on it in your spare time once you fulfill the requirements. 👍🏻 😃 

_Check the comments under the review._




## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding! 👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by peterdtitan** [comment]
Made new changes and fixed the consistency of the text elements. 

Screenshot 2022-06-25 at 1 45 09 PM

File: about.html [hunk 2]
```diff
@@ -0,0 +1,282 @@
...
+            ></iframe>
+          </div>
+          <div>
+            <iframe
+              src="https://www.youtube.com/watch?v=-Qp0a4jG23A"
+              title="CHOGM Stream"
+              allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture"
+              allowfullscreen
+            ></iframe>
+          </div>
```
Comment (aimemalaika) [inline about.html:251]: - [x] Kindly make sure that the youtube iframe wark or you can consider removing them
Screen Shot 2022-06-25 at 13 22 31

**Review by aimemalaika — APPROVED** [review]
Hi @peterdtitan  & @zainsadaqat 

In my opinion, your project is ready for presentation!  There is nothing else to say other than... it's time to merge it :shipit: 

133591850-fdcb7dd3-dd7a-4274-81e7-0e59434148f7

Good luck! 🎉 


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**